docs/ai/pi-chat.md

```
@@ -84,34 +83,37 @@ Plane AI is accessible in two ways:
+::: info
 Creating items requires [Build mode](#build-mode).
 :::
 
 #### Create a work item
 
 1. Ensure you have a project selected using the Focus selector.
 2. Type a creation command describing what you want to create:
-    ```
-      Create a bug report for the login page crash
-    ```
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)